src/components/src/utils/ConfigProvider.ts

```diff
@@ -110,21 +112,10 @@ export interface IEpConfig {
 
 interface IConfig {
   config: IEpConfig | any,
-  intl: any
-}
-
-let config: IConfig = {
-  config: {},
-  intl: { get: str => str },
-};
-
-export function init(_config: IConfig) {
-  return new Promise((resolve) => {
-    config = Object.freeze(_config);
-    resolve({ version: packageJson.version });
-  });
```

Comment:
Why wouldn't we want to freeze our configuration object here? It should be a constant that is immutable. Will protect against any developers accidentally changing its value and causing unexpected behaviour

Reply:
If we decide we do want to freeze these settings, maybe we can have a getIntl() function that is exported as well.

Reply:
I've added Object.freeze to config (https://github.com/elasticpath/react-pwa-reference-storefront/pull/651/files#diff-1d1e22969e225d881fdb8f6a28b77ba7R117) but I would'n worry much about this since the whole function is being moved to React context in the second part of this refactoring.